spelling_wordlist.txt

```diff
@@ -34,6 +34,7 @@ bugfix
 bytecode
 bytecodes
 callee
+certora
```

Comment:
weird this is necessary...

Reply:
it complains about `https://www.certora.com/...`